gazebo_ros_control:
  pid_gains:
    joint1: { p: 1000.0,  d: 10.0, i: 0.01 }
    joint2: { p: 1000.0,  d: 10.0, i: 0.01 }
    joint3: { p: 1000.0,  d: 10.0, i: 0.01 }
    joint4: { p:  500.0,  d: 10.0, i: 0.01 }
    joint5: { p:  100.0,  d: 10.0, i: 0.01 }
    joint6: { p:  100.0,  d: 10.0, i: 0.01 }

    